src/tests/flask/base_flask_setup_test.py

```diff
@@ -0,0 +1,33 @@
+from abc import abstractmethod
+
+from flask import Flask
+from src.core.sqlalchemy_config import get_engine
+from src.flask.api_setup import setup_api
+from src.lib.entities.sqlalchemy_orm_mapping import Base
+from src.tests.base_env_config_test import BaseEnvConfigTest
+
+engine = get_engine()
```

Comment:
Do this in the setup method. No need to do this outside of the class

Reply:
done.